.github/workflows/pr-test.yml

```diff
@@ -56,8 +56,11 @@ jobs:
         with:
           go-version-file: 'go.mod'
       
-      - name: Set up Docker
-        uses: docker/setup-buildx-action@v2
+      - name: Install Dagger CLI
+        run: |
+          cd /usr/local
+          curl -L https://dl.dagger.io/dagger/install.sh | sh
+          dagger version
```

Comment:
Is there a reason to do this vs using the Dagger GitHub action? If installing the binary directly is preferred, I wonder if we could/should move this to the `make` file.